test/jdk/com/sun/management/ThreadMXBean/ThreadAllocatedMemory.java

```diff
@@ -201,38 +206,102 @@ private static void testGetThreadsAllocatedBytes()
             try {
                 threads[i].join();
             } catch (InterruptedException e) {
-                System.out.println("Unexpected exception is thrown.");
+                System.out.println("Unexpected exception thrown.");
+                e.printStackTrace(System.out);
+                break;
+            }
+        }
+    }
+
+    private static void testGetAllThreadAllocatedBytes()
+        throws Exception {
+
+        // baseline should be positive
+        Thread curThread = Thread.currentThread();
+        long cumulative_size = mbean.getAllThreadAllocatedBytes();
```

Comment:
Java style for variables is camel-case, `cumulativeSize`.

Reply:
Fixed.